pkg/preflights/embed.go

```diff
@@ -3,20 +3,29 @@ package preflights
 import (
 	"context"
 	_ "embed"
+	"fmt"
 
 	"github.com/replicatedhq/troubleshoot/pkg/apis/troubleshoot/v1beta2"
 	"github.com/replicatedhq/troubleshoot/pkg/loader"
 )
 
-//go:embed host-preflight.yaml
-var clusterHostPreflightYAML string
+var (
+	//go:embed host-preflight.yaml
+	clusterHostPreflightYAML string
+)
```

Comment:
which one is it 😆 
(you changed from multiline to single line in pkg/defaults/defaults.go)